test/upgrade/upgrade.go

```diff
@@ -78,3 +83,19 @@ func createNewService(serviceName string, t *testing.T) {
 	url := resources.Service.Status.URL.URL()
 	assertServiceResourcesUpdated(t, clients, names, url, test.PizzaPlanetText1)
 }
+
+func CreateTestNamespace() pkgupgrade.Operation {
+	return pkgupgrade.NewOperation("CreateTestNamespace", func(c pkgupgrade.Context) {
+		createTestNamespace(c.T, "serving-tests")
```

Comment:
You'll want to use the value from the flag here:
https://github.com/knative/serving/blob/a61884652e0169ec11acea25161120afb51988b9/test/e2e_flags.go#L79-L80

Reply:
Right. Thanks.

Reply:
Pushed.